app/pages/lab/project-details.cjsx

```diff
@@ -110,6 +119,15 @@ module.exports = React.createClass
         </div>
 
         <div className="column">
+          {if @props.project.links?.organization
+            <div>
+              {if @state.organization
+                <p>This project is part of the <Link to={"/organizations/#{@state.organization.slug}"}>{@state.organization.display_name}</Link> organization.</p>
+              else
+                <p>This project is linked to <strong>Organization #{@props.project.links.organization}</strong>.</p>}
```

Comment:
In the `if` condition, organization is lower case, and in the `else` condition, it's upper case - maybe good to be consistent with either upper or lower case in both?